extensions/wikia/PortableInfobox/services/PortableInfoboxDataService.class.php

```diff
@@ -113,13 +113,32 @@ protected function getInfoboxes( $text ) {
 	}
 
 	/**
-	 * @desc returns the text from inside of the first <includeonly> tag
+	 * @desc returns the text from inside of the first <includeonly> tag and
+	 * without the nowiki and pre tags.
 	 * @param $text string template text
-	 * @return mixed
+	 * @return string
 	 */
 	protected function getIncludeonlyText( $text ) {
 		preg_match_all( "/<includeonly>(.+)<\/includeonly>/sU", $text, $result );
 
-		return isset( $result[1][0] ) ? $result[1][0] : false;
+		if ( !isset( $result[1][0] ) ) {
+			return null;
```

Comment:
Previously you were returning false here, right now it's null. Is there a reason for this change? Will the code behave the same way?